### ISSUE TITLE
Can't edit content fields in iOS

### ISSUE THREAD

**Issue by jmchat** [verbb/vizy#76]
**Description**
Content fields within Vizy can't be edited on iOS. You can't select or add any text in a content field. Only the title fields work on iOS. 

**Steps to reproduce**
1. Take an iPhone or iPad and visit the control panel and try to edit a text in a content field.

**Additional info**
- Plugin version: 1.0.7
- Craft version: Craft Pro 3.7.16
- Multi-site: Yes

**Additional context**
- Tested on iOS 15 -> iPhone and iOS 12 -> iPad mini

**Comment by engram-design** [comment]
This is a known issue, covered in https://github.com/verbb/vizy/issues/17#issuecomment-867409322 and https://github.com/verbb/vizy/issues/58

**Comment by iuscare** [comment]
Hey, first of all Happy Christmas! I was wondering if this will be fixed with https://github.com/verbb/vizy/issues/58#issuecomment-997755670 too?

**Comment by engram-design** [comment]
@iuscare At first glance, this does seem to also be fixed. Let me know if you're running into issues on your end with Safari!

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.9](https://github.com/verbb/vizy/releases/tag/1.0.9)
